src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2507,6 +2507,62 @@ Plain\ Text=Plain Text
+Document\ splitter\ -\ chunk\ size=Document splitter - chunk size
+Document\ splitter\ -\ overlap\ size=Document splitter - overlap size
+Document\ splitter\ chunk\ size\ must\ be\ greater\ than\ 0=Document splitter chunk size must be greater than 0
+Document\ splitter\ overlap\ size\ must\ be\ greater\ than\ 0\ and\ less\ than\ chunk\ size=Document splitter overlap size must be greater than 0 and less than chunk size
+Embedding\ model=Embedding model
+Embeddings\ generation=Embeddings generation
+Expert\ settings=Expert settings
+Leave\ these\ fields\ as\ is,\ if\ you\ are\ not\ sure\ of\ their\ purpose.=Leave these fields as is, if you are not sure of their purpose.
+Only\ PDF\ files\ are\ supported.=Only PDF files are supported.
+OpenAI\ model=OpenAI model
+Please\ provide\ a\ non-empty\ and\ unique\ citation\ key\ for\ this\ entry.=Please provide a non-empty and unique citation key for this entry.
+Please\ wait=Please wait
+RAG\ -\ maximum\ results\ count=RAG - maximum results count
+RAG\ -\ minimum\ score=RAG - minimum score
+RAG\ max\ results\ count\ must\ be\ greater\ than\ 0=RAG max results count must be greater than 0
+RAG\ min\ score\ must\ be\ greater\ than\ 0\ and\ less\ than\ 1=RAG min score must be greater than 0 and less than 1
+Regenerate\ embeddings\ cache=Regenerate embeddings cache
+Regenerate\ embeddings\ cache\ for\ current\ library?=Regenerate embeddings cache for current library?
+Regenerating\ embeddings\ cache...=Regenerating embeddings cache...
+Reset\ expert\ settings\ to\ default=Reset expert settings to default
+Temperature=Temperature
+Temperature\ must\ be\ between\ 0\ and\ 2=Temperature must be between 0 and 2
+The\ embeddings\ of\ the\ file\ are\ currently\ being\ generated.\ Please\ wait,\ and\ at\ the\ end\ you\ will\ be\ able\ to\ chat.=The embeddings of the file are currently being generated. Please wait, and at the end you will be able to chat.
+These\ parameters\ affect\ how\ AI\ will\ answer\ you\ questions.=These parameters affect how AI will answer you questions.
+Are\ you\ sure\ you\ want\ to\ clear\ the\ chat\ history\ with\ this\ entry?=Are you sure you want to clear the chat history with this entry?
+Context\ window\ size=Context window size
+Context\ window\ size\ must\ be\ greater\ than\ 0=Context window size must be greater than 0
+Embeddings\ generation\ for\ %0=Embeddings generation for %0
+In\ order\ to\ use\ AI\ chat,\ set\ OpenAI\ API\ key\ inside\ JabRef\ preferences\ (AI\ tab)=In order to use AI chat, set OpenAI API key inside JabRef preferences (AI tab)
+In\ order\ to\ use\ AI\ chat,\ you\ need\ to\ enable\ chatting\ with\ attached\ PDF\ files\ in\ JabRef\ preferences\ (AI\ tab)=In order to use AI chat, you need to enable chatting with attached PDF files in JabRef preferences (AI tab)
+Instruction\ for\ AI\ (also\ known\ as\ prompt\ or\ system\ message)=Instruction for AI (also known as prompt or system message)
+The\ OpenAI\ token\ has\ to\ be\ provided=The OpenAI token has to be provided
```

Comment:
"An OpenAI token has to be provided" -- there is more than one token available. Therefore, no "the"